crates/rolldown/tests/esbuild/default/import_forms_with_minify_identifiers_and_no_bundle/bypass.md

```diff
@@ -1,5 +1,6 @@
 # Reason
 1. Could be done in minifier
+2. Rolldown not try to merge external default import binding
```

Comment:
Is this a typo? Seems this PR is gonna to support merging external default import binding.

Reply:
Yeah, it is typo